blueprints/docs-page/index.js

```diff
@@ -98,7 +96,7 @@ function writeRoute(action, name, options) {
 
 function updateDocsTemplate(options) {
   let routeName = options.entity.name;
-  let docsTemplatePath = options.pods
+  let docsTemplatePath = options.pod
```

Comment:
Found this wee bug while I was writing tests. I guess I missed it with my manual testing (I'm old and senile...)